extensions/amp-accordion/amp-accordion.md

```diff
@@ -127,79 +133,68 @@ animation when the content is expanded and "roll up" animation when collapsed.
 
 [/example]
 
-[filter formats="websites"]
-
-### disable-session-states
-
-Include the `disable-session-states` attribute on `<amp-accordion>` to disable
-collapsed/expanded state preservation.
-
-[/filter] <!-- formats="websites" -->
-
 ### expanded
 
 Apply the `expanded` attribute to a nested `<section>` to expand that section when the page loads.
 
-### expand-single-section
-
-Apply the `expand-single-section` attribute to `amp-accordion` to specify that
-only one `<section>` can expand at a time. If the user clicks or taps on a
-collapsed `<section>`, any currently expanded `<section>` collapses.
+Bind the `expanded` attribute to a `<section>` to expand or collapse that
+section. An expanded section collapses if the expression evaluates as false. A
+collapsed section expands if the expression evaluates as anything other than
+false.
 
-[example preview="top-frame" playground="true"]
+[example preview="top-frame" playground="true" imports="amp-accordion:1.0"]
 
 ```html
-<amp-accordion expand-single-section>
-  <section>
+<amp-accordion>
+  <section
+    [expanded]="sectionOne"
+    on="expand:AMP.setState({sectionOne: true});collapse:AMP.setState({sectionOne: false})"
+  >
     <h2>Section 1</h2>
-    <p>Content in section 1.</p>
+    <p>Bunch of awesome content</p>
   </section>
   <section>
     <h2>Section 2</h2>
-    <div>Content in section 2.</div>
+    <div>Bunch of awesome content</div>
   </section>
-  <section>
+  <section expanded>
     <h2>Section 3</h2>
-    <amp-img
-      src="{{server_for_email}}/static/inline-examples/images/squirrel.jpg"
-      width="320"
-      height="256"
-    ></amp-img>
+    <div>Bunch of awesome content</div>
   </section>
 </amp-accordion>
+<button on="tap:AMP.setState({sectionOne: true})">Expand section 1</button>
+<button on="tap:AMP.setState({sectionOne: false})">Collapse section 1</button>
 ```
 
 [/example]
 
-### [data-expand]
+### expand-single-section
 
-Bind the `[data-expand]` attribute to a `<section>` to expand or collapse that
-section. An expanded section collapses if the expression evaluates as false. A
-collapsed section expands if the expression evaluates as anything other than
-false.
+Apply the `expand-single-section` attribute to `amp-accordion` to specify that
+only one `<section>` can expand at a time. If the user clicks or taps on a
+collapsed `<section>`, any currently expanded `<section>` collapses.
```

Comment:
```suggestion
Allow only one section to expand at a time by applying the `expand-single-section` attribute to the `<amp-accordion>` element. This means if a user taps on a collapsed `<section>`, it will expand and close other open `<section>`'s.
```

Reply:
Thanks, I used something similar in the latest revision.  Replaced "close" and "open" with "collapsed" and "expanded" in the last sentence.